abnormal_security/plugin.spec.yaml

```diff
@@ -0,0 +1,227 @@
+      title: Summary Insights
+      description: Summary insights
+      type: '[]string'
+      required: false
+    remediationTimestamp:
+      title: Remediation Timestamp
+      description: Remediation timestamp
+      type: string
+      required: false
+    isRead:
+      title: Is Read
+      description: Is Read
+      type: boolean
+      required: false
+    attackedParty:
+      title: Attacked Party
+      description: Attacked party
+      type: string
+      required: false
+    abxMessageId:
+      title: ABX Message ID
+      description: ABX Message ID
+      type: integer
+      required: false
+    abxPortalUrl:
+      title: ABX Portal URL ID
+      description: ABX portal URL ID
+      type: string
+      required: false
+    subject:
+      title: Subject
+      description: Subject
+      type: string
+      required: false
+    fromAddress:
+      title: From Address
+      description: From address
+      type: string
+      required: false
+    fromName:
+      title: From Name
+      description: From name
+      type: string
+      required: false
+    toAddresses:
+      title: To Address
+      description: To address
+      type: string
+      required: false
+    recipientAddress:
+      title: Recipient Address
+      description: Recipient address
+      type: string
+      required: false
+    receivedTime:
+      title: Received Time
+      description: Received time
+      type: string
+      required: false
+    sentTime:
+      title: Sent Time
+      description: Sent time
+      type: string
+      required: false
+    internetMessageId:
+      title: Internet Message ID
+      description: Internet message ID
+      type: string
+      required: false
+    urls:
+      title: URLs
+      description: URLs
+      type: '[]string'
+      required: false
+  threat_details:
+    threatId:
+      title: Threat ID
+      description: Threat ID
+      type: string
+      required: false
+    messages:
+      title: Messages
+      description: List of messages
+      type: '[]message'
+      required: false
+connection:
+  hostname:
+    title: Host
+    description: Abnormal Security host URL
+    type: string
+    required: true
+    example: https://api.abnormalplatform.com
+    default: https://api.abnormalplatform.com
+  api_key:
+    type: credential_secret_key
+    title: Abnormal Security API Key
+    description: Abnormal Security API Key
+    required: true
+    example: Vud5CDO9ULIV3SJOcp
+actions:
+  get_threats:
+    title: Get Threats
+    description: Get a list of up to 100 threats identified in the Abnormal Security Threat Log, if no input filter dates are provided, it will return up to 100 latest results
+    input:
+      from_date:
+        title: From Date
+        description: This input enables you to filter your results from a certain date, the date has to be in ISO 8601 format - YYYY-MM-DDTHH:MM:SSZ
+        type: string
+        required: false
+        example: 2021-03-01T21:11:38Z
```

Comment:
@jmcadams-r7 @mrinehart-r7 is there a date format that we need the user to input to match the product's date picker?

Reply:
@jmcadams-r7 @mrinehart-r7 bump